client/client_test.go

```diff
@@ -45,3 +45,39 @@ func TestClient_retryHTTP(t *testing.T) {
 		t.Fatalf("Expected 5 retries, got: %d", httpmock.GetTotalCallCount())
 	}
 }
+
+func TestClient_userAgent(t *testing.T) {
+	httpmock.Activate()
```

Comment:
Looks like it is used already but why the choice of httpmock over stdlib [httptest](https://golang.org/pkg/net/http/httptest/)?

Reply:
I just did a little copypasta from the previous test for this test to make it easier to write

Reply:
It looks like our old guy preferred that to httptest